#! /bin/sh /usr/share/dpatch/dpatch-run
## 03_revert_chr_translate.dpatch by Chris Lamb <chris@chris-lamb.co.uk>
##
## DP: Revert a2cc8caa17243ef239f17b1cdbac9cae913d23de from upstream, which
## DP: seems to be the cause of FTBFS on arm and armel.

@DPATCH@

diff -urNad pl-5.6.58.orig/packages/chr/chr_translate.chr pl-5.6.58/packages/chr/chr_translate.chr
--- pl-5.6.58.orig/packages/chr/chr_translate.chr	2008-07-25 19:47:20.000000000 +0100
+++ pl-5.6.58/packages/chr/chr_translate.chr	2008-07-25 20:47:01.000000000 +0100
@@ -8432,7 +8432,7 @@
 				( NewMask == 0 ->
 					del_attr(Var,Mod)
 				;
-					setarg(1,TAttr,NewMask),
+					arg(1,TAttr,NewMask),
 					setarg(Position1,TAttr,NewSusps)
 				)
 			;
